editor/src/components/inspector/fill-hug-fixed-control.tsx

```diff
@@ -743,6 +744,21 @@ export const allElementsAreGroupChildren = createSelector(
   },
 )
 
+export const allElementsArePositionedAbsolutelySelector = createSelector(
+  metadataSelector,
+  selectedViewsSelector,
+  (metadata, selectedViews): boolean => {
+    if (selectedViews.length === 0) {
+      return false
+    }
+    return selectedViews.every((path) => {
```

Comment:
We have `strictEvery` that abstracts away this check (just an idea, it's up to you whether you want to use it or not)

Reply:
love it, I did not know it existed! Updated the code, including the other selector right above the new one.